dev-env/bin/da-ghci

```diff
@@ -88,11 +101,9 @@ def main():
     except subprocess.CalledProcessError:
         sys.exit(1)
 
-
```

Comment:
Why the whitespace change? [PEP 8](https://www.python.org/dev/peps/pep-0008/#blank-lines) says to

> Surround top-level function and class definitions with two blank lines.

Reply:
My editor settings automatically rewrite three consecutive newline characters into two, on save. I can revert this!

Reply:
Reverted